frontend/packages/dev-console/integration-tests/testData/monitoring/monitoring-cluster-monitoring-config.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    enableUserWorkload: true
```

Comment:
Need to add new line

Reply:
We can add this YAMLs into testData. The PR with YAMLs folder is ready to merge. Once the PR is merged, you can add these YAMLs to YAMLs folder in testData.